poi-ooxml/src/main/java/org/apache/poi/xssf/usermodel/XSSFRow.java

```diff
@@ -120,18 +121,14 @@ public XSSFSheet getSheet() {
     }
 
     /**
-     * Alias for {@link #cellIterator()} to allow  foreach loops:
-     * <blockquote><pre>
-     * for(Cell cell : row){
-     *     ...
-     * }
-     * </pre></blockquote>
+     * Cell spliterator over the physically defined cells
      *
-     * @return an iterator over cells in this row.
+     * @return a spliterator over cells in this row.
      */
     @Override
-    public Iterator<Cell> iterator() {
```

Comment:
add back the original iterator

Reply:
I've moved it to the `Row` parent interface, so its essentially the same (alias for `cellIterator()`), see https://github.com/apache/poi/pull/290/files#diff-41fe318702760f407b9cfab2d3e3ceb3693f848c2255e3e5c993a0d7b4832624